scripts/nerves-env-helper.sh

```diff
@@ -110,9 +110,9 @@ export CROSSCOMPILE
 export REBAR_PLT_DIR=$NERVES_SDK_SYSROOT/usr/lib/erlang
 export CC=$CROSSCOMPILE-gcc
 export CXX=$CROSSCOMPILE-g++
-export CFLAGS="-D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -pipe -O2 -I$NERVES_SDK_SYSROOT/usr/include"
-export CPPFLAGS="-D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -I$NERVES_SDK_SYSROOT/usr/include"
-export CXXFLAGS="-D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -pipe -O2 -I$NERVES_SDK_SYSROOT/usr/include"
+export CFLAGS="-D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -pipe -O2 --sysroot $NERVES_SDK_SYSROOT/usr/include"
```

Comment:
```suggestion
export CFLAGS="-D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -pipe -O2 --sysroot $NERVES_SDK_SYSROOT"
```
Shouldn't just be the sysroot path to match what is happening in the Elixir file as well?

Reply:
Yes, omg, thanks for the catch.